### PR TITLE
Remove try-with-resource statements (#969 Part B)

### PR DESCRIPTION
**Pull request by RoiEXLab**
This resolves the warning, in which the timer variable is not being used inside the try block.
@DanVanAtta we can discuss this here

---
This change is [Reviewable](https://reviewable.io/reviews/triplea-game/triplea/974)

### CODE REVIEW

File: src/games/strategy/performance/Perf.java [hunk 1]
```diff
@@ -1,56 +0,0 @@
...
-
-import java.util.prefs.Preferences;
-
-
-/**
- * Provides a high level API to the game engine for performance measurements.
- * This class handles the library details and sends output to 'PerformanceConsole.java'
- */
-// TODO Move the methods in this class to PerfTimer
-public class Perf {
```
Comment (RoiEXLab) [inline src/games/strategy/performance/Perf.java]: @DanVanAtta the merging of those classes was a TODO task...

**Comment by RoiEXLab** [comment]
@DanVanAtta Could we focus on the GameRunner thing in another PR?

File: src/games/strategy/performance/Perf.java (hunk 1 above)
Comment (DanVanAtta) [inline src/games/strategy/performance/Perf.java]: sure enough.  It looks to still be auto-closeable now. Will review tomorrow, after a quick look things looked okay.

Comment (RoiEXLab) [inline src/games/strategy/performance/Perf.java]: I readded the interface...

File: src/games/strategy/performance/PerfTimer.java [hunk 2]
```diff
@@ -1,19 +1,32 @@
...
  * Provides a high level API to the game engine for performance measurements.
  * This class handles the library details and sends output to 'PerformanceConsole.java'
  */
-public class PerfTimer {
+public class PerfTimer implements Closeable {
 
-  protected static final PerfTimer DISABLED_TIMER = new PerfTimer("disabled");
+  private static final PerfTimer DISABLED_TIMER = new PerfTimer("disabled");
 
   public final String title;
```
Comment (DanVanAtta) [inline src/games/strategy/performance/PerfTimer.java]: Instance variables should be declared after static.

File: src/games/strategy/performance/PerfTimer.java [hunk 3]
```diff
@@ -64,11 +68,16 @@ public static PerfTimer startTimer(String title) {
...
     long stopMicros = stopNanos / 1000;
 
     long milliFraction = (stopMicros % 1000) / 100;
     long millis = (stopMicros / 1000);
     PerformanceConsole.getInstance().append(millis + "." + milliFraction + " ms - " + perfTimer.title + "\n");
   }
-}
\ No newline at end of file
+
+  @Override
```
Comment (DanVanAtta) [inline src/games/strategy/performance/PerfTimer.java]: Could you group this method closer to the 'stop' method, this way related entities/methods are closer together.

Comment (DanVanAtta) [inline src/games/strategy/performance/PerfTimer.java]: Second note, we probably do not need a 'stop' method, can be merged up with the close method IMO

**Comment by RoiEXLab** [comment]
@DanVanAtta updated!

**Pull request merged**